android问题：


20200921
android L2分笔和L2逐笔接口
现状：自动化平台运行无结果，日志报异常TestcaseException（java.util.ConcurrentModificationException），
本地(android+python)运行一条用例没有问题，多条用例报异常TestcaseException（java.util.ConcurrentModificationException），
直接android工程中调用接口没有问题（相当于一条用例）
分析：欧阳反应是并发写冲突的异常，L2提交结果代码的问题（可能是递归导致的）
问题：目前不知道怎么解决


Time: 67.784
There were 2 failures:
1) requestWork(com.chi.ssetest.cases.L2TickDetailV2Test_1)
com.chi.ssetest.TestcaseException
	at com.chi.ssetest.cases.L2TickDetailV2Test_1.requestWork(L2TickDetailV2Test_1.java:73)
	at java.lang.reflect.Method.invoke(Native Method)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:764)
2) requestWork(com.chi.ssetest.cases.L2TickDetailV2Test_1)
com.chi.ssetest.TestcaseException
	at com.chi.ssetest.cases.L2TickDetailV2Test_1.requestWork(L2TickDetailV2Test_1.java:73)
	at java.lang.reflect.Method.invoke(Native Method)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:764)
	
	
	异常打印输出
	java.util.ConcurrentModificationException